### DISCUSSION TITLE
NuGet package is very old and shows security issues

### DISCUSSION THREAD

**Discussion by ghiscoding** [6pac/SlickGrid#785] [General]
@6pac I saw that 5 new security issues are showing in the security section and they all seem to be pointing to the very old NuGet package that you added years ago and that didn't get updated in a while, can that file be deleted?

https://github.com/6pac/SlickGrid/blob/c6a58296792f92ae8e494bda7499859e7fffcc50/slickgrid.nuspec#L27-L30

![image](https://github.com/6pac/SlickGrid/assets/643976/d75bba98-9da5-4536-8672-e0a88a71e371)

**Comment by 6pac** [comment]
Sure. Can we delete the whole .nuspec file? If it's required then we should update the authors to include you, remove jQuery from the dependencies and perhaps add SortableJS? I think that's now the only mandatory external dependency.

**Reply by ghiscoding** [reply]
no need to add my name, just yours is enough. My thinking is that this NuGet file isn't used much and can probably be deleted since it's so old and nobody complained

**Comment by 6pac** [comment]
isn't this the spec file for the nuget package? ie. this sets the metadata about the package itself.

i think the issue is that the jquery dependency that was listed was for a very old version of jquery and hadn't been updated to a recent version (of course, now we don't need it at all)

**Reply by ghiscoding** [reply]
I was just reporting the security issues, how you deal with it is up to you since I have no use for this NuGet package. I thought of removing it because it seems old, but this decision is all up to you. Go ahead with what you think is the best approach. Cheers
